src/main/java/io/nats/client/support/JwtUtils.java

```diff
@@ -216,6 +218,9 @@ public static String issueJWT(NKey signingKey, String publicUserKey, String name
     public static class UserClaim extends NatsClaim<UserClaim> {
         public Permission pub;
         public Permission sub;
+        public int subs = NO_LIMIT;
```

Comment:
I believe it's worth considering the use of a long data type instead of int in this particular context. This decision depends on the server type and the potential range of values it may encounter. By using a long, we can ensure compatibility and accommodate larger values, thus future-proofing our code. It's always wise to analyze the specific requirements and consider the potential scalability of our systems.